syncd/VendorSai.cpp

```diff
@@ -675,7 +675,7 @@ sai_status_t VendorSai::bulkRemove(
             ptr = m_apis.lag_api->remove_lag_members;
             break;
 
-        case SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER:
+        case SAI_OBJECT_TYPE_NEXT_HOP_GROUP:
             ptr = m_apis.next_hop_group_api->remove_next_hop_group_members;
```

Comment:
this should be SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER

Reply:
ok, I reverted this commit